fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -967,14 +981,18 @@ ACTOR Future<Void> assignMutationsToStorageServers(CommitBatchContext* self) {
 				}
 				self->toCommit.writeTypedMessage(m);
 
-				auto& teams = pProxyCommitData->keyInfo[m.param1].teams;
-				self->pTeamMessageBuilder->write(m, teams);
+				auto& storageTeams = pProxyCommitData->keyInfo[m.param1].storageTeams;
```

Comment:
Not required for this PR, the `keyInfo`, i.e., `ServerCacheInfo.storageTeams` are not initialized. Probably after #5059 is merged.